### PR TITLE
Fix Translation

### PR DESCRIPTION
**Pull request by moon-anti**
-update in line 318 change simply to secara sederhana
-delete line 346, which contain  \nhandle bisa menjadi Id pengguna atau username pengguna
-change administrator to Administrator in several line
-change name to nama and fedid to ID Federasi
-change user_id to ID Pengguna in information so users will not confuse
-change reply to balas pesan

### CODE REVIEW

**Comment by moon-anti** [comment]
Ready to merge.

**Comment by moon-anti** [comment]
Done, ready to merge.

File: language/id.yml [hunk 1]
```diff
@@ -23,11 +23,11 @@ back-button: ⬅️ Kembali
...
-  - <code>__italic__</code>: Buat dua garis bawah di antara kata, akan menghasilkan <i>italic</i> text.
-  - <code>**bold**</code>: Buat dua tanda bintang di antara kata, akan menghasilkan <b>bold</b> text
-  - <code>`code`</code>: Buat tanda kutip terbalik atau backticks di antara kata, akan menghasilkan <code>monospace</code> text.
-  - <code>~~strike~~</code>: Buat tanda gelombang atau tildes di antara kata, akan menghasilkan <del>strikethrough</del> text
-  - <code>--underline--</code>: Buat tanda penghubung di antara kata, akan menghasilkan  <u>underline</u> text
+  - <code>__italic__</code>: Buat dua garis bawah di antara kata, akan menghasilkan <i>italic</i> teks.
+  - <code>**bold**</code>: Buat dua tanda bintang di antara kata, akan menghasilkan <b>bold</b> teks.
+  - <code>`code`</code>: Buat tanda kutip terbalik atau backticks di antara kata, akan menghasilkan <code>monospace</code> teks.
+  - <code>~~strike~~</code>: Buat tanda gelombang atau tildes di antara kata, akan menghasilkan <del>strikethrough</del> teks.
+  - <code>--underline--</code>: Buat tanda penghubung di antara kata, akan menghasilkan <u>underline</u> teks.
```
Comment (MrMissx) [inline language/id.yml:30]: This should be `teks ........`
E.G: `teks <i>italic</i>`.

**Review by MrMissx — CHANGES_REQUESTED** [review]

File: language/id.yml (hunk 1 above)
Comment (moon-anti) [reply language/id.yml]: Done, thankyou.

**Review by MrMissx — APPROVED** [review]

**Pull request merged**